test/lib/render-helpers.js

```diff
@@ -38,5 +39,7 @@ function mountWithRouter (node) {
     childContextTypes: { router: shape({}), t: () => {} },
   })
 
-  return mount(node, createContext())
+  return mount(<BrowserRouter>
```

Comment:
Small nitpick: can we wrap this is parens? (I'm trying to get our JSX formatting into a better state, one where we are consistent about the indentation, wrapping multiple lines, where we place props, etc. etc. e.g. #7592.)

Something like so:

```js
  return mount((
    <BrowserRouter>
      {node}
    </BrowserRouter>
  ), createContext())
```

Reply:
I'm actually not sure this violates `react/jsx-wrap-multilines`. That rule looks for specific types of expressions, and I don't see this one listed. Which would kinda make sense - it's already wrapped in the parens from `mount`, so it can't possibly fall victim to the semi-colon insertion that is the motivation for requiring parens elsewhere.

Reply:
Well the spirit of that PR is to enforce this wrapping too, since I personally believe this is easier to read. (I am under the impression that the combination of the rules there would enforce this but I'd need to confirm it.)

Reply:
How about something like?

```diff
diff --git a/test/lib/render-helpers.js b/test/lib/render-helpers.js
index 5f7a921d5..bdcc8a5a2 100644
--- a/test/lib/render-helpers.js
+++ b/test/lib/render-helpers.js
@@ -39,7 +39,11 @@ function mountWithRouter (node) {
     childContextTypes: { router: shape({}), t: () => {} },
   })
 
-  return mount(<BrowserRouter>
-    {node}
-  </BrowserRouter>, createContext())
+  const Wrapper = () => (
+    <BrowserRouter>
+      {node}
+    </BrowserRouter>
+  )
+
+  return mount(<Wrapper />, createContext())
 }
```

Reply:
That works too!